src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py

```diff
@@ -799,10 +807,11 @@ def _find_missing_deps(self, compile_failure_log, target):
         self.context.log.info(suggestion_msg)
 
       if no_suggestions:
-        self.context.log.debug('Unable to find any deps from target\'s transitive '
+        self.context.log.error('Unable to find any deps from target\'s transitive '
                                'dependencies that provide the following missing classes:')
         no_suggestion_msg = '\n   '.join(sorted(list(no_suggestions)))
-        self.context.log.debug('  {}'.format(no_suggestion_msg))
+        self.context.log.error('  {}'.format(no_suggestion_msg))
```

Comment:
When these two messages are concatenated, they will be a bit redundant:
```
Unable to find any deps from target's transitive dependencies that provide the following missing classes:
  com.foo.Bar
Missing-deps-suggest could not find the dependencies needed to compile your target. Please see https://www.pantsbuild.org/3rdparty_jvm.html for more information.
```

It's possible that this should just template in a URL so that the overall wording of the message could be more cohesive?

Reply:
You're right, it is a bit redundant. I can change it to just the second sentence. I didn't want to only be able to configure the URL though because I think it should be more configurable, primarily for internal use.